core/templates/components/forms/custom-forms-directives/thumbnail-display.component.ts

```diff
@@ -53,14 +55,16 @@ export class ThumbnailDisplayComponent implements OnInit, OnChanges {
     } else {
       this.imageSourceInView = this.imgSrc;
     }
-    this.width = this.aspectRatio === '4:3' ? '248px' : '320px';
-    this.height = this.aspectRatio === '4:3' ? '186px' : '180px';
+    this.width = this.aspectRatio === '4:3' ? '248px' : '100px';
+    this.height = this.aspectRatio === '4:3' ? '186px' : '100px';
   }
 
   ngOnChanges(): void {
-    this.updateSvgInViewIfSafe();
-    this.width = this.aspectRatio === '4:3' ? '248px' : '320px';
-    this.height = this.aspectRatio === '4:3' ? '186px' : '180px';
+    if (this.imgSrc !== undefined) {
+      this.updateSvgInViewIfSafe();
+      this.width = this.aspectRatio === '4:3' ? '248px' : '100px';
+      this.height = this.aspectRatio === '4:3' ? '186px' : '100px';
+    }
```

Comment:
These values are causing it to look disproportionate, a separate issue have been also opened for this thing https://github.com/oppia/oppia/issues/12264

Reply:
Please undo the changes to these values. This will be fixed in a separate PR by @lkbhitesh07.

Reply:
Could we make changes in the HTML instead such that, if the imgSrc is undefined, we use a hardcoded height / width and if not, use the dynamic one?